routing/chainview/interface_test.go

```diff
@@ -445,17 +499,36 @@ func testFilterSingleBlock(node *rpctest.Harness, chainView FilteredChainView,
 // testFilterBlockDisconnected triggers a reorg all the way back to genesis,
 // and a small 5 block reorg, ensuring the chainView notifies about
 // disconnected and connected blocks in the order we expect.
-func testFilterBlockDisconnected(node *rpctest.Harness,
-	chainView FilteredChainView, chainViewInit chainViewInitFunc,
-	t *testing.T) {
+func testFilterBlockDisconnected(t *testing.T,
+	chainViewInit chainViewInitFunc) {
+
+	// Initialize the harness around a btcd node which will serve as our
+	// dedicated miner to generate blocks, cause re-orgs, etc. We'll set up
+	// this node with a chain length of 125, so we have plenty of BTC to
+	// play around with.
+	node := unittest.NewMiner(t, netParams, []string{"--txindex"}, true, 25)
```

Comment:
nit: extract this code block into a function?

Reply:
done!